x/downtime-detector/genesis.go

```diff
@@ -0,0 +1,48 @@
+package downtimedetector
```

Comment:
I believe we still need to add tests for these methods for this module. Its okay if we don't address it here, but should make an issue if not.

Edit: Ah looks like you did this in keeper_test.go. Should we move them to a genesis_test.go file?

Reply:
did it